### ISSUE TITLE
Progress component doesn't read as expected in VO

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-react#4241]
**Describe the issue. What is the expected and unexpected behavior?**

Every example in the  progress component is currently reading as "indeterminate progress bar" even though there is a Title prop passed inside of the component. The experience is different in Core, where it is reading the correct title and the percentage value, which are both absent here. 

I discovered this in the Insights Dashboard where we have passed a Title prop, and the percentage value, yet it is still reading as "indeterminate progress bar".

Hopefully we can update the progress component to fix this, or have a short-term fix in the meantime.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
- go to https://patternfly-react.surge.sh/documentation/react/components/progress
- look at the first progress bar
- turn on VO
- use the roter to hear what is read out loud when tabbing through the progress bar

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**

Bug

**What is your product and what release version are you targeting?**

Insights Dashboard.

**Comment by redallen** [comment]
Is this now after #4248 ?

**Comment by jessiehuff** [comment]
Closed by #4248
